pom.xml

```diff
@@ -253,306 +161,40 @@
-                <version>${spring-cloud.version}</version>
-            </dependency>
-
-            <dependency>
-                <groupId>org.springframework</groupId>
-                <artifactId>spring-context</artifactId>
-                <version>${spring.version}</version>
-            </dependency>
-
-
-            <dependency>
-                <groupId>com.netflix.hystrix</groupId>
-                <artifactId>hystrix-core</artifactId>
-                <version>${hystrix.version}</version>
-            </dependency>
-
-            <dependency>
-                <groupId>com.netflix.archaius</groupId>
-                <artifactId>archaius-core</artifactId>
-                <version>${archaius.version}</version>
-            </dependency>
-
-            <dependency>
-                <groupId>com.netflix.ribbon</groupId>
-                <artifactId>ribbon-core</artifactId>
-                <version>${ribbon-version}</version>
-            </dependency>
-
-            <dependency>
-                <groupId>com.netflix.ribbon</groupId>
-                <artifactId>ribbon</artifactId>
-                <version>${ribbon-version}</version>
-            </dependency>
-
-            <dependency>
-                <groupId>com.netflix.ribbon</groupId>
-                <artifactId>ribbon-httpclient</artifactId>
-                <version>${ribbon-version}</version>
-            </dependency>
-
-            <dependency>
-                <groupId>com.netflix.ribbon</groupId>
-                <artifactId>ribbon-loadbalancer</artifactId>
-                <version>${ribbon-version}</version>
-            </dependency>
-
-            <dependency>
-                <groupId>io.reactivex</groupId>
-                <artifactId>rxjava</artifactId>
-                <version>${rxjava.version}</version>
-            </dependency>
-
-            <dependency>
-                <groupId>io.reactivex</groupId>
-                <artifactId>rxjava-reactive-streams</artifactId>
-                <version>${rxjava-reactive-streams.version}</version>
-            </dependency>
-
-            <!--Rx2 part-->
-            <dependency>
-                <groupId>io.projectreactor.addons</groupId>
-                <artifactId>reactor-adapter</artifactId>
-                <version>${reactor-adapter.version}</version>
-            </dependency>
-
-            <dependency>
-                <groupId>io.reactivex.rxjava2</groupId>
-                <artifactId>rxjava</artifactId>
-                <version>${rxjava2.version}</version>
-            </dependency>
-
-            <!--Jetty part-->
-            <dependency>
-                <groupId>org.eclipse.jetty</groupId>
-                <artifactId>jetty-reactive-httpclient</artifactId>
-                <version>${jetty-reactive-httpclient.version}</version>
-            </dependency>
-
+            <!-- Other -->
             <dependency>
                 <groupId>io.kptfh.reactivejson</groupId>
```

Comment:
io.kptfh.reactivejson is this still required?

Reply:
yes